### ISSUE TITLE
[Feature] Improvement Request for Getting Started documentation

### ISSUE THREAD

**Issue by lechnerc77** [bitprj/Intro-To-Serverless#438]
### Use-cases

[https://github.com/bitprj/Intro-To-Serverless/blob/main/GETTING_STARTED.md#question-can-i-run-a-check-on-my-own-computer](https://github.com/bitprj/Intro-To-Serverless/blob/main/GETTING_STARTED.md#question-can-i-run-a-check-on-my-own-computer)

### Proposal

The referenced section in the documentation states that 

>> This is not required, as you can always commit your code to the counselor bot to check, but is definitely good to know.

I would like to see an addition to that as the flow for a developer interacting with GitHub usually is:

1. Make your code changes
2. Test the changes locally (1 and 2 are usually done in a circle or even the other way round when it come to Test-Driven Development )
3. Push the changed code to your fork
4. Do a pull request and let the checks and tests run in the repo. 

Bottom line: setting up your local development in a way that you can do the checks first locally and get feedback there is normal and the preferred way. Would be appreciated to have a statement on that in the text.    

### References

[https://github.com/bitprj/Intro-To-Serverless/blob/main/GETTING_STARTED.md#question-can-i-run-a-check-on-my-own-computer](https://github.com/bitprj/Intro-To-Serverless/blob/main/GETTING_STARTED.md#question-can-i-run-a-check-on-my-own-computer)

**Comment by lechnerc77** [comment]
@emsesc @ganning127 : Can we add the `pinned` label to the issues that should be done after the summer session so that they are ignored by the stale bot?
